vyper/builtin_functions/functions.py

```diff
@@ -820,6 +820,10 @@ def _storage_element_getter(index):
     return LLLnode.from_list(["sload", ["add", "_sub", ["add", 1, index]]], typ=BaseType("int128"),)
 
 
+def _calldata_element_getter(index):
+    return LLLnode.from_list(["calldataload", ["mul", 32, index]], typ=BaseType("int128"))
```

Comment:
I think the widest type possible --  BaseType("uint256") -- should be used here as int128 is no longer the default in vyper.